docs/source/developers/guide/resources.rst

```diff
@@ -55,7 +55,7 @@ Additional information
 
   It is possible to install the Arrow R package from the nightly builds which are daily development
   builds of the R package and are not the official releases. See more on the
-  `Install R package article <https://ursalabs.org/arrow-r-nightly/articles/install.html#install-the-nightly-build>`_.
+  `Install R package article <https://arrow.apache.org/docs/r/articles/install.html#install-the-nightly-build>`_.
```

Comment:
Previously this pointed to the nightly docs, which I guess makes sense because this is about the nightly packages and the installation instructions for those may change, well, nightly. Is it worth keeping the nightly docs URL?

```suggestion
  `Install R package article <https://arrow.apache.org/docs/dev/r/articles/install.html#install-the-nightly-build>`_.
```